api.py

```diff
@@ -427,6 +528,19 @@ def ingest_entities(dataset_id, data, **kwargs):
     return response
 
 
+def _ingest_entity_set(dataset_id, entity_set, **kwargs):
```

Comment:
did you also want `ingest_entities` to call this?

Reply:
Yes